pyirf/interpolation/component_estimators.py

```diff
@@ -212,39 +217,61 @@ def __init__(
             grid_points,
         )
 
-        if not isinstance(bin_edges, np.ndarray):
-            raise TypeError("Input bin_edges is not a numpy array.")
-        elif not isinstance(bin_contents, np.ndarray):
+        # Ignore bin_edges if both inter- and extrapolator are NearestNeighborSeacher
```

Comment:
mmh... this breaks somehow the separation of concerns... and it's pretty weird to ignore a required, positional argument